src/meta/src/controller/streaming_job.rs

```diff
@@ -544,6 +545,40 @@ impl CatalogController {
         Ok(())
     }
 
+    pub async fn build_cancel_command(
```

Comment:
Could you add some docs here to elaborate on the cancellation procedure for sink-into-table?